PelotonEppSdk/Models/Document.cs

```diff
@@ -18,13 +18,13 @@ public class Document
         /// <summary>
         /// Name that the document has when represented on a file system.
         /// </summary>
-        [Required, StringLength(128, ErrorMessage = "The " + nameof(Filename) + " field must be less than 128 characters long.")]
+        [Required, StringLength(128, ErrorMessage = "The " + nameof(Filename) + " field must be less than 128 characters in length.")]
         public string Filename { get; set; }
 
         /// <summary>
         /// Media Type of the Document. Example http://en.wikipedia.org/wiki/Internet_media_type.
         /// </summary>
-        [Required, StringLength(256, ErrorMessage = "The " + nameof(MediaType) + " field must be less than 256 characters long.")]
+        [Required, StringLength(256, ErrorMessage = "The " + nameof(MediaType) + " field must be less than 256 characters in length.")]
```

Comment:
Shouldn't this be "must be 256 or fewer characters in length" in be 1) Actually correct and 2) Consistent with the other similar validation messages.